linalg/hypre.hpp

```diff
@@ -396,6 +409,13 @@ class HypreParMatrix : public Operator
    /// Returns the transpose of *this
    HypreParMatrix * Transpose() const;
 
+   /** Returns principle submatrix given by array of indices of connections
+       with relative size > \@ threshold in *this. */
+#if MFEM_HYPRE_VERSION >= 21800
+   HypreParMatrix * ExtractSubmatrix(Array<int> &indices,
+                                     double threshhold=0.0) const;
```

Comment:
The parameter `indices` is an input parameter (read-only) so it is best to add a `const` qualifier to its type.